.gitignore

```diff
@@ -11,3 +11,6 @@ build/
 #*
 *.html
 *.log
+*.journal
+citest.egg-info/
+venv/
```

Comment:
You dont need to .git-ignore it. Just dont add it to begin with.
If this is a problem, perhaps add *.egg-info since none of it would go into git anyway.